### PR TITLE
Backup in iCloud

### PR DESCRIPTION
**Pull request by Hirobreak**

### CODE REVIEW

File: iOS-Email-Client/Controllers/BackupViewController.swift [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+            DBManager.update(account: self.myAccount, frequency: selected)
+            self.toggleOptions()
+            BackupManager.shared.checkAccounts()
+        }
+        self.presentPopover(popover: pickerPopover, height: 295)
+    }
+}
+
+enum BackupFrequency: String {
+    case minute = "Minute"
```
Comment (danieltigse) [inline iOS-Email-Client/Controllers/BackupViewController.swift]: isn't it localized?

File: iOS-Email-Client/Controllers/BackupViewController.swift [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let cell = tableView.dequeueReusableHeaderFooterView(withIdentifier: "headerCell") as! BackupHeaderView
+        cell.setContent(email: myAccount.email, isUploading: uploading, lastBackupDate: lastBackupDate, lastBackupSize: lastBackupSize)
+        cell.progressLabel.text = processMessage
+        cell.progressView.setProgress(progress, animated: true)
+        return cell
+    }
+    
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return (uploading || lastBackupDate != nil) ? 180 : 120
```
Comment (danieltigse) [inline iOS-Email-Client/Controllers/BackupViewController.swift]: magic numbers

File: iOS-Email-Client/Libs/Env.swift [hunk 3]
```diff
@@ -77,7 +77,7 @@ struct Env {
         return Locale.current.languageCode ?? "en"
     }
     
-    static let databaseVersion: UInt64 = 17
+    static let databaseVersion: UInt64 = 19
```
Comment (danieltigse) [inline iOS-Email-Client/Libs/Env.swift]: and 18?

File: iOS-Email-Client/Views/es-419.lproj/RestoreUIPopover.strings [hunk 4]
```diff
@@ -0,0 +1,12 @@
+
+/* Class = "UIButton"; normalTitle = "Restore from Cloud"; ObjectID = "CDQ-w1-eqI"; */
+"CDQ-w1-eqI.normalTitle" = "Restaurar desde iCloud";
+
+/* Class = "UILabel"; text = "Something about how to restore from Drive  you should give us permissions, and you can always restore later.\n\n"; ObjectID = "WNQ-UH-w8G"; */
```
Comment (danieltigse) [inline iOS-Email-Client/Views/es-419.lproj/RestoreUIPopover.strings]: don't forget this translation

File: iOS-Email-Client/Controllers/BackupViewController.swift (hunk 1 above)
Comment (Hirobreak) [reply iOS-Email-Client/Controllers/BackupViewController.swift]: Yes, but not here

File: iOS-Email-Client/Views/es-419.lproj/RestoreUIPopover.strings (hunk 4 above)
Comment (Hirobreak) [reply iOS-Email-Client/Views/es-419.lproj/RestoreUIPopover.strings]: It's a placeholder, string is overwritten in code

**Pull request merged**